spec/rubocop/cli/auto_gen_config_spec.rb

```diff
@@ -394,6 +394,97 @@ def f
                 ''].join("\n"))
     end
 
+    context 'when --only is used' do
+      before do
+        create_file('example.rb', <<~RUBY)
+          # frozen_string_literal: true
+
+          class MyClass
+            def initialize
+              p "No documentation for class"
+            end
+          end
+
+          def f
+          #{'  #' * 46}
+            if #{'a' * 120}
+              return y
+            end
+
+            z
+          end
+        RUBY
+      end
+
+      context 'when --only does not contain Layout/Linelength' do
+        it 'generates TODO only for the mentioned COP and Layout/LineLength' do
```

Comment:
There's not need to capitalize COP like you've done. I also don't get what's the special significance of `LineLength` here if it's wasn't passed an argument.

Reply:
I will update the wording used here to be more straight forward - sorry for this

Reply:
updated descriptions